tensornetwork/network_components.py

```diff
@@ -534,6 +534,34 @@ def get_tensor(self):
   def set_tensor(self, tensor):
     self.tensor = tensor
 
+  def __add__(self, other: Union[int, float, "BaseNode"]) -> "BaseNode":
+    if not hasattr(self, '_tensor'):
+      raise AttributeError("Please provide a valid tensor for this Node.")
+
+    if isinstance(other, BaseNode):
+      other_tensor = other.get_tensor()
```

Comment:
I thought we had deprecated get_tensor()?

Reply:
Replaced with tensor.